src/csv2docx/csv2docx.py

```diff
@@ -4,21 +4,21 @@
 from mailmerge import MailMerge
 
 
-def create_output_folder(path: str) -> Path:
+def create_output_folder(output_path: str) -> Path:
     """Creates a path to store output data if it does not exists.
     Args:
         path: the path from user in any format (relative, absolute, etc.)
     Returns:
         A path to store output data.
     """
-    path = Path(path)
+    path = Path(output_path)
```

Comment:
Had to rename the `path` parameter because mypy indicated that `path` did not have methods such as `.exists()`. This seems like a failing with mypy, but felt that it was appropriate since it makes it easier to read.

Reply:
I fully agree (and clear variable names are good practise)